### DISCUSSION TITLE
Move translation activity to Crowdin

### DISCUSSION THREAD

**Discussion by alroniks** [audacity/audacity#3989] [Ideas & Feature Requests]
What do you think about improving translation process and moving translations into https://crowdin.com/
It is totally free for open source projects. And, this is pretty important, it is much easier to bring translators, who not always tech aware people, but they know language aspects well, via such translating system.

**Comment by LWinterberg** [comment]
What are the benefits of it over Transifex? That's what currently is being used by the community (https://explore.transifex.com/klyok/audacity/) and also is what our sister project is using: https://explore.transifex.com/musescore/musescore/

**Reply by alroniks** [reply]
I didn't find any information about Transifex on this page https://www.audacityteam.org/community/translators/ why I suggested to use Crowdin. But if it exists already, that's nice!
